src/feat/pitch-functions.cc

```diff
@@ -774,12 +774,20 @@ int32 OnlinePitchFeatureImpl::NumFramesAvailable(
   if (!input_finished_)
     frame_length += nccf_last_lag_;
   if (num_downsampled_samples < frame_length) return 0;
-  else
-    if (input_finished_ && !snip_edges) {
-      return (int32)(num_downsampled_samples * 1.0f / frame_shift + 0.5f);
+  else {
```

Comment:
Hi Dan,

You are right, it actually helps a lot to use cpplint; although since most of the existing code is "dirty" anyway, it is not always easy to figure out what to clean.

Anyway, I passed cpplint on all the code I modified, and now all the modified parts should hopefully be in an acceptable state. Thanks!